cdserver/Gemfile

```diff
@@ -25,6 +25,8 @@ gem 'puma', '~> 3.0'
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
```

Comment:
Please add a feature test for /authentication/login.feature   
As a user I want to authenticate to the system so that I can be certain unauthorized persons cannot access the system.
```
Scenario: successful user login
Given I have opened capdash2
When I login as the following user
      | username        | password |
      | joeblow@hbo.com | secret   |
Then I should receive the following response
  | status | 200 |
  | token | $validJwtToken |
```

```
Scenario: unsuccessful user login, wrong password
Given I have opened capdash2
When I login as the following user
      | username        | password |
      | joeblow@hbo.com | iforgot   |
Then I should receive the following response
  | status | 401 Unauthorized |
```

Note that this will require you to refactor and unify the previous Then statements
`Then I should receive the HTTP response`
`Then I should receive the response`
Let's make it just one that works for both-- it includes both the HTTP status code and an optional payload.